compatibility.bs

```diff
@@ -199,6 +203,25 @@ be performant.)
 
 <h2 id="css-compat-section">CSS Compatibility</h2>
 
+<h3 id="css-cssom-compat">CSS Object Model</h3>
+
+<code>-webkit-</code> prefixed CSS properties must be available for getting and setting as <a>camel-cased attribute</a> attributes of the {{CSSStyleDeclaration}} interface.
+
+<div class="example">
+For example, it must be possible to get and set a <code>-webkit-transform</code> property value for a reference to a given element <code>elm</code> via <code>elm.style.webkitTransform</code>.
+</div>
+
+<code>-webkit-</code> prefixed CSS properties must be available for getting and setting as <a>dashed attribute</a> attributes of the {{CSSStyleDeclaration}} interface.
+
+<div class="example">
+For example, it must be possible to get and set a <code>-webkit-transform</code> property value for a reference to a given element <code>elm</code> via <code>elm.style[-webkit-transform]</code>.
```

Comment:
Don't use must in examples. (also above.)

Should be `elm.style["-webkit-transform"]`